### PR TITLE
Refine scripts/README for ae reproducibility verifier

### PR DESCRIPTION
**Pull request by yzhu53**

### CODE REVIEW

File: linux/reproducibility/ae_reproducibility_verifier/README.md [hunk 1]
```diff
@@ -1,100 +1,62 @@
...
+# How to Verify Intel(R) Prebuilt AE Reproducibility
 
 ## Purpose
----
-Since Intel(R) prebuilt AEs have been signed by Intel's private key, it's hard to verify AE reproduciblity by comparing your local AE builds with Intel AEs using "diff" command.  
-So this doc will give an approach using your local AE builds to verify Intel(R) prebuilt AEs are:
 
-* Built using same code and toolsets that being opensourced
-* Signed using same config.xml that being opensourced
+Since Intel(R) prebuilt AEs have been signed by Intel's private key, it's not easy to verify your AE builds against Intel(R) Prebuilt AEs in a common way (e.g. comparing the file checksum).
```
Comment (jbdelcuv) [inline linux/reproducibility/ae_reproducibility_verifier/README.md:5]: common way sounds strange to me.

File: linux/reproducibility/ae_reproducibility_verifier/README.md [hunk 2]
```diff
@@ -1,100 +1,62 @@
...
 
 ## Purpose
----
-Since Intel(R) prebuilt AEs have been signed by Intel's private key, it's hard to verify AE reproduciblity by comparing your local AE builds with Intel AEs using "diff" command.  
-So this doc will give an approach using your local AE builds to verify Intel(R) prebuilt AEs are:
 
-* Built using same code and toolsets that being opensourced
-* Signed using same config.xml that being opensourced
+Since Intel(R) prebuilt AEs have been signed by Intel's private key, it's not easy to verify your AE builds against Intel(R) Prebuilt AEs in a common way (e.g. comparing the file checksum).  
+So the doc gives an approach to guarantee Intel(R) prebuilt AEs are:
```
Comment (jbdelcuv) [inline linux/reproducibility/ae_reproducibility_verifier/README.md:6]: This document outlines a process to guarantee that Intel(R) prebuilt AEs are:

File: linux/reproducibility/ae_reproducibility_verifier/README.md [hunk 3]
```diff
@@ -1,100 +1,62 @@
...
-  ```
-    $ {SGX_SDK}/bin/x64/sgx_sign sign -enclave user_build_unsigned_ae -key user_private_key -config intel_ae_config_xml -out user_signed_ae
-  ```
-3) Dump the whole enclave_metadata of user_signed_a, and then extract partial enclave_metadata to "user_metadata.txt" excluding below fields:  
-    "enclave_css.header.date, enclave_css.enclave_css.key.modulus, enclave_css.key.exponent, enclave_css.key.signature, enclave_css.buffer.q1, enclave_css.buffer.q2"
+- <a id="materials">Prepare below 4 materials</a>
+  - **intel_signed_ae**  
+ It is the Intel(R) prebuilt AE to be verified, which can be downloaded from <https://download.01.org/intel-sgx/latest/linux-latest/> and <https://download.01.org/intel-sgx/latest/dcap-latest/linux/>.
+  - **user_build_unsigned_ae**  
+ The unsigned AE is built by yourself in a SGX docker container by following the [reproducibility README.md](../README.md).
```
Comment (jbdelcuv) [inline linux/reproducibility/ae_reproducibility_verifier/README.md:23]: The unsigned AE built...

**Review by jbdelcuv — COMMENTED** [review]
Minor suggestions.

File: linux/reproducibility/ae_reproducibility_verifier/README.md [hunk 4]
```diff
@@ -1,100 +1,62 @@
-Verify Intel(R) Prebuilt AE Reproducibility:
-================================
----------
+# How to Verify Intel(R) Prebuilt AE Reproducibility
```
[the remark this answers is not in the document]
Comment (yzhu53) [reply linux/reproducibility/ae_reproducibility_verifier/README.md:1]: Opened https://github.com/intel/linux-sgx/pull/992 to address

File: linux/reproducibility/ae_reproducibility_verifier/README.md (hunk 1 above)
Comment (yzhu53) [reply linux/reproducibility/ae_reproducibility_verifier/README.md:5]: Opened https://github.com/intel/linux-sgx/pull/992 to address

File: linux/reproducibility/ae_reproducibility_verifier/README.md (hunk 3 above)
Comment (yzhu53) [reply linux/reproducibility/ae_reproducibility_verifier/README.md:23]: Opened https://github.com/intel/linux-sgx/pull/992 to address

File: linux/reproducibility/ae_reproducibility_verifier/README.md (hunk 2 above)
Comment (yzhu53) [reply linux/reproducibility/ae_reproducibility_verifier/README.md:6]: Opened https://github.com/intel/linux-sgx/pull/992 to address

**Pull request merged**